### ISSUE TITLE
ssl error in registry.py

### ISSUE THREAD

**Issue by shouro** [GluuFederation/gluu-flask#83]
In node/node.py when it calls get_registry_cert(), I am getting this error,
```
exceptions.TypeError: getsockaddrarg: AF_INET address must be tuple, not list
```
I try to fix, get_registry_cert() function in registry.py and pass ('registry.gluu.org','5000')
in get_registry_cert(), I get this error,
```
exceptions.TypeError: an integer is required
```
then I make '5000' a int and pass ('registry.gluu.org', 5000) into get_registry_cert()
and i get this error,
```
(ssl.SSLError: [Errno 1] _ssl.c:510: error:1408F10B:SSL routines:SSL3_GET_RECORD:wrong version number)
```

**Comment by iromli** [comment]
Likely we have different underlying SSL version. I will also take a look on this one.

**Comment by iromli** [comment]
There's a similar command that's not relying on Python SSL implementation:

```sh
echo -n | openssl s_client -connect registry.gluu.org:5000 | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' > registry.crt
```

Perhaps we can run it using subprocess.

**Comment by iromli** [comment]
@shouro please try new codebase; let me know if that fixes the issue in your machine

**Comment by shouro** [comment]
closed #83
